internal/install_exists.go

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package txqualitychecks
+
+import "os"
+
+// InstallExists defines a struct for
+type InstallExists struct {
+}
+
+// NewInstallExists func return a new check based on QualityGuideline interface.
+func NewInstallExists() QualityGuideline {
+	return &InstallExists{}
+}
+
+// Name implements ExternalDescription from interface
```

Comment:
I like, that you add docs for the functions/methods, but in this particular case, I would rather move it to the interface itself, since the intention is defined by the interface. The implementations then do not need to add it again and again, since `Name()` `Description` and `ExternalDescription()` do not really have specific implementation details that have to be described on top of the interface intend

Reply:
Jap, you're right. To be honest, I was pretty unsure about where to add docs. I'm going to change this.

Reply:
Done